documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -214,6 +229,7 @@ include::modules/non-admin-permissions-for-ui.adoc[leveloffset=+2]
 [id="migrating-virtual-machines_{context}"]
 === Migrating virtual machines
 
+<<<<<<< HEAD
```

Comment:
there's a merge-conflict that wasn't properly resolves here

Reply:
Strange; GitHub didn't call attention to that. Good eye!